extensions/types/vscode-mssql.d.ts

```diff
@@ -730,6 +739,66 @@ declare module 'vscode-mssql' {
 		deploymentOptions?: DeploymentOptions;
 	}
 
+	//#region ISqlProjectsService
+
+	//#region Parameters
+
+	export interface SqlProjectParams {
```

Comment:
Please update this in the VS Code MSSQL repo too.

Reply:
Will do.